geopandas/tests/test_geom_methods.py

```diff
@@ -321,7 +321,7 @@ def test_is_empty(self):
 
     def test_is_ring(self):
         expected = Series(np.array([True] * len(self.g1)), self.g1.index)
-        self._test_unary_real('is_ring', expected, self.g1)
+        self._test_unary_real('is_ring', expected, self.g1.exterior)
```

Comment:
instead of this, we could maybe also call `.exterior` in the `is_ring` defintion on GeoSeries to keep previous behaviour?